src/lib/orionld/temporal/temporalTenantInitialise.cpp

```diff
@@ -244,7 +257,7 @@ bool temporalTenantInitialise(const char* tenant)
             "created_at TIMESTAMP NOT NULL,"
             "modified_at TIMESTAMP NOT NULL,"
             "observed_at TIMESTAMP NOT NULL,"
-            "PRIMARY KEY (entity_id,id,observed_at,created_at,modified_at))",
+            "PRIMARY KEY (instance_id))",
```

Comment:
I imagine having the entity and timestamps as part of the primary key make lookups faster.
If we remove that, we might lose performance (instanceId is not something people will use for lookups).
Let's discuss this.